maps/submaps/engine_submaps/engine.dm

```diff
@@ -14,7 +14,7 @@
 	allow_duplicates = FALSE
 
 /datum/map_template/engine/rust
-	name = "R-UST Engine"
+	name = "EngineSubmap_RUST"
```

Comment:
this is required because config entries have their key/values separated by spaces, it's possible to do it the other way around but i'd like to avoid it